src/node_platform.cc

```diff
@@ -0,0 +1,201 @@
+#include "node_platform.h"
+
+namespace node {
+
+using v8::Isolate;
+using v8::Platform;
+using v8::Task;
+using v8::TracingController;
+
+static void FlushTasks(uv_async_t* handle) {
+  NodePlatform* platform = static_cast<NodePlatform*>(handle->data);
+  platform->FlushForegroundTasksInternal();
+}
+
+static void DeleteHandle(uv_handle_t* handle) {
+  delete handle;
```

Comment:
This doesn't look right, it deletes `sizeof(uv_handle_t)` but the handles are `uv_async_t` and `uv_timer_t` instances.  I'd pass lambdas at the `uv_close()` call sites that do the right thing.